x-pack/plugins/security_solution/scripts/endpoint/agent_emulator/services/endpoint_response_actions.ts

```diff
@@ -118,6 +116,7 @@ export const sendEndpointActionResponse = async (
   action: ActionDetails,
   { state }: { state?: 'success' | 'failure' } = {}
 ): Promise<LogsEndpointActionResponse> => {
+  const endpointActionGenerator = new EndpointActionGenerator();
```

Comment:
Why create an instance of the generator class every time? Seems unnecessary in this context (CLI tool)